content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -291,9 +291,9 @@ sandboxing implementations are available:
 
 * [gVisor](https://gvisor.dev/) intercepts syscalls from containers and runs them through a
   userspace kernel, written in Go, with limited access to the underlying host.
-* [Kata Containers](https://katacontainers.io/) is an OCI compliant runtime that allows you to run
+* [Kata Containers](https://katacontainers.io/) provide a secure container runtime that allows you to run
   containers in a VM. The hardware virtualization available in Kata offers an added layer of
-  security for containers running untrusted code.
+  security for containers running untrusted code. [v3.0.2](https://github.com/kata-containers/kata-containers/releases/tag/3.0.2) and above support the OCI Runtime Specification.
```

Comment:
This statement may require some evidence as I'm unsure about its validity.

The older release (https://github.com/kata-containers/kata-containers/releases/tag/2.5.2) mentions

> Kata Containers 2.5.2 support the OCI Runtime Specification [v1.0.0-rc5](https://github.com/opencontainers/runtime-spec/releases/tag/v1.0.0-rc5)


💭 Could we potentially omit this level of detail from here? People interested in the OCI compliance status of Kata Containers could easily check the Kata Containers website (https://katacontainers.io/), conveniently linked earlier in the text. What do you think?

Reply:
I agree. This is addressed in the latest commit. Thanks @dipesh-rawat.